contracts/voting_escrow/src/contract.rs

```diff
@@ -553,6 +599,207 @@ fn withdraw(deps: DepsMut, env: Env, info: MessageInfo) -> Result<Response, Cont
     }
 }
 
+/// ## Description
+/// Sets a max exit penalty and a slashed funds receiver. Can be called by the owner only.
+fn configure_early_withdrawal(
+    deps: DepsMut,
+    info: MessageInfo,
+    max_exit_penalty: Decimal,
```

Comment:
Let's keep it as optional here to allow changing only slashed_fund_receiver

Reply:
Sure, fixed